### PR TITLE
Reader Stream Refresh: turn off follow button on cards for Blog Stream and Feed Stream

### LINKED ISSUE #9228: Cards on Site View should not have Follow links
For example: https://wpcalypso.wordpress.com/read/feeds/52296202

When cards appear on a Site View page, the cards should not have the Follow link. (No need, because it's at the top of the page, and is just repetitive on the cards.)

### PR DESCRIPTION
**Pull request by bluefuton**
Fixes #9228.

Before:

screen shot 2016-11-14 at 15 17 30

After:

screen shot 2016-11-14 at 15 17 36

### To test

Ensure that the Follow button is absent from cards in blog streams:

http://calypso.localhost:3000/read/blogs/76295396

...and feed streams:

http://calypso.localhost:3000/read/feeds/8901148

...but make sure it still shows up in Following.

### CODE REVIEW

File: client/reader/controller.js [hunk 1]
```diff
@@ -164,7 +163,7 @@ module.exports = {
 					key: 'following',
 					listName: i18n.translate( 'Followed Sites' ),
 					store: followingStore,
-					showPrimaryFollowButtonOnCards: false,
+					showPrimaryFollowButtonOnCards: true,
```
Comment (samouri) [inline client/reader/controller.js]: was this false before?

**Comment by samouri** [comment]
LGTM 🚢

File: client/reader/controller.js [hunk 2]
```diff
@@ -292,7 +293,7 @@ module.exports = {
 						analyticsPageTitle,
 						mcKey
 					),
-					showPrimaryFollowButtonOnCards: false,
+					showPrimaryFollowButtonOnCards: true,
```
Comment (blowery) [inline client/reader/controller.js]: shouldn't this be false, everything in the a8c stream is from sites you already follow

File: client/reader/controller.js (hunk 1 above)
Comment (blowery) [inline client/reader/controller.js]: should be false too

**Comment by blowery** [comment]
Fixed the two cases where follow buttons started showing up incorrectly. Will land this after tests pass.

**Pull request merged**